src/components/CardGrid/CardGrid.tsx

```diff
@@ -61,7 +62,9 @@ function CardGrid({
   }, [playlist.feedid]);
 
   const renderTile = (playlistItem: PlaylistItem) => {
-    const { mediaid, title, duration, seriesId, episodeNumber, seasonNumber, shelfImage } = playlistItem;
+    const { mediaid, title, duration, episodeNumber, seasonNumber, shelfImage } = playlistItem;
```

Comment:
I think it's also probably time to just pass the whole media item as a generic key value blob into the Card component (https://github.com/jwplayer/exp-ott-web-app/blob/main/src/components/Card/Card.tsx#L36). There's definitely getting to be too much series specific content and it would be nice if those things only were accessed inside the card component where we actually know it's a series.

Reply:
I agree that it is better to pass the whole item. For now we already have series checks it in 3 places:

<img width="480" alt="Screenshot 2023-04-18 at 11 36 34" src="https://user-images.githubusercontent.com/27254986/232736736-dbb377c7-61b2-4e44-88f9-ce1f1e0c4d73.png">

Reply:
Ya, let's pass the whole item. The number of params on this component are getting unwieldy.